### DISCUSSION TITLE
Ability to set Display Name in shortcut

### DISCUSSION THREAD

**Discussion by neilsapno** [citizenfx/fivem#2382] [Engineering Discussion]
Roleplay servers nowadays requires people to change their Steam name or Display name inside FiveM to their character's name. By adding an option to change the name in the executable shortcut can be a good QOL update, especially for people who plays on multiple servers and has different names for each server.

Something like `-displayName Name Here`

It can be very useful if you already use multiple shortcuts that direct connects to a server then sets the name at the same time.

**Comment by blattersturm** [comment]
A more sensible implementation could be something like Discord's 'server profiles'.

**Reply by neilsapno** [reply]
That would be much harder to code for sure and too much for its purpose. Servers only reads the Player name in the settings and other than that just the steam and rockstar token which isn't modifiable.

**Reply by blattersturm** [reply]
A launch option would be hard to discover, which is a pain that plays out over the long term. 

Also, a launch option could be interpreted in various ways - should it persist? should it be a temporary override? should this be a `-` parameter or a `+set`? should there be an option to add a temporary override to `fivem://` links or `cfx.re/join/` URLs in the browser? - so even that in and of itself isn't too simple either, discounting documentation and discoverability concerns as well.

Similarly, your concerns about identifiers are also incomplete - there's stuff such as the "license:" token having two different sources (and servers not using "license2:"), or players potentially wanting to hide their Steam account, Discord account (or use multiple Discord accounts!), or other accounts for privacy reasons.

**Reply by neilsapno** [reply]
I get your point now. I just hope someone can add the feature.
